package.json

```diff
@@ -81,5 +81,6 @@
     "watch:react": "npm run --workspace packages/react build:watch",
     "watch:storybook": "npm run --workspace storybook start",
     "watch:tokens": "npm run --workspace proprietary/tokens watch"
-  }
+  },
+  "packageManager": "pnpm@9.12.1"
```

Comment:
Thanks for the PR! Seems like a fine addition to me. 

One question: [the docs strongly recommend adding a hash here](https://github.com/nodejs/corepack?tab=readme-ov-file#when-authoring-packages), any reason not to?

Reply:
IIRC @matijs found that the hash could vary per platform, so the hash for one version would be different for different for various CPU architectures, or something like that.